multi-node/aws/README.md

```diff
@@ -6,14 +6,30 @@ View the full instructions at https://coreos.com/kubernetes/docs/latest/kubernet
 
 ### Download pre-built binary
 
+First, make sure you have imported the [CoreOS Application Signing Public Key](https://coreos.com/security/app-signing-key/).
+
+```sh
+gpg --keyserver pgp.mit.edu --recv-key FC8A365E
+
+#Validate the key fingerprint
+gpg2 --fingerprint FC8A365E
+# Key fingerprint = 18AD 5014 C99E F7E3 BA5F  6CE9 50BD D3E0 FC8A 365E
+```
+
+Go to the [releases](https://github.com/coreos/coreos-kubernetes/releases) and download the latest release tarball and detached signature (.sig) for your architecture.
+
 ```sh
 PLATFORM=linux-amd64
 # Or
 PLATFORM=darwin-amd64
 
-wget https://coreos-kubernetes.s3.amazonaws.com/kube-aws/latest/${PLATFORM}/kube-aws
-chmod +x kube-aws
-# Add kube-aws binary to your PATH
+#Verify GPG signature
+gpg --default-key FC8A365E --verify kube-aws-${PLATFORM}-amd64.tar.gz.sig kube-aws-${PLATFORM}-amd64.tar.gz
```

Comment:
Can drop the --default-key. gpg ignores it and just ensures the signature is valid. Also maybe use gpg2, but that's just my preference.